beacon-chain/blockchain/receive_attestation.go

```diff
@@ -96,6 +97,11 @@ func (s *Service) VerifyLmdFfgConsistency(ctx context.Context, a *ethpb.Attestat
 	return s.verifyLMDFFGConsistent(ctx, a.Data.Target.Epoch, a.Data.Target.Root, a.Data.BeaconBlockRoot)
 }
 
+// VerifyFinalizedConsistency verifies input root is consistent with finalized store.
+func (s *Service) VerifyFinalizedConsistency(ctx context.Context, r []byte) error {
+	return s.verifyFinalizedConsistency(ctx, r)
+}
```

Comment:
Why have a public method that is just passthrough to the private method?

Reply:
I was following `VerifyLmdFfgConsistency`'s pattern but it was a bad follow. I'll fix it